chore(minifier): remove duplicate minifier test

camc314 opened the pull request:
these tests already exist below (and have been enabled)
https://github.com/oxc-project/oxc/blob/34fe7c00a393471309d336fcdcfef86402a9c8dc/crates/oxc_minifier/src/ast_passes/peephole_remove_dead_code.rs#L576-L597

i suspect this was left in as a copy/pasta error

camc314 commented:
* **#6792** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6792?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @camc314 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

Boshen commented:
### Merge activity

* **Oct 22, 11:39 PM EDT**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

camc314 committed:
chore(minifier): remove duplicate minifier test (#6792)

these tests already exist below (and have been enabled)
https://github.com/oxc-project/oxc/blob/34fe7c00a393471309d336fcdcfef86402a9c8dc/crates/oxc_minifier/src/ast_passes/peephole_remove_dead_code.rs#L576-L597

i suspect this was left in as a copy/pasta error
